modules/monitoring/application/forms/Command/Object/AcknowledgeProblemCommandForm.php

```diff
@@ -88,7 +88,7 @@ public function createElements(array $formData = array())
         $expire = isset($formData['expire']) ? $formData['expire'] : $acknowledgeExpire;
         if ($expire) {
             $expireTime = new DateTime();
-            $expireTime->add(new DateInterval('PT1H'));
+            $expireTime->add(new DateInterval($config->get('settings', 'acknowledge_expire_time', 'PT1H')));
```

Comment:
Hello @mapa82 and thank you for your suggestion!

however, I'd like you to make this option configurable via GUI.

Best,
AK

Reply:
As discussed w/ @lippserd (offline):

No, this isn't required (yet).